src/main/steps/common/respondent-summary/form.njk

```diff
@@ -47,7 +47,7 @@
             <div class="govuk-form-group">
           <div class="govuk-checkboxes govuk-!-padding-bottom-2">
             <div class="govuk-checkboxes__item">
-              <input class="govuk-checkboxes__input" style="background-color: white" id="declarationCheck" name="declarationCheck" type="checkbox" value="declaration">
+              <input class="govuk-checkboxes__input" style="background-color: white" id="declarationCheckInput" name="declarationCheck" type="checkbox" value="declaration">
```

Comment:
Why is the id changed? Is the id used in code any where? Also you have not associated the input and label

Reply:
name and for are associated right?
should it be id and for ?